scipy/stats/tests/test_morestats.py

```diff
@@ -757,6 +757,22 @@ def test_wilcoxon_bad_arg():
     assert_raises(ValueError, stats.wilcoxon, [1], [1,2])
     assert_raises(ValueError, stats.wilcoxon, [1,2], [1,2], "dummy")
 
+def test_wilcoxon_arg_type():
+    # Should be able to accept list as arguments.
+    arr = [1, 2, 3, 0, -1, 3]
+    
+    T, p = stats.wilcoxon(arr, zero_method="pratt")
+    assert_allclose(T, 2.5)
+    assert_allclose(p, 0.091690281549429153)
+
+    T, p = stats.wilcoxon(arr, zero_method="zsplit")
+    assert_allclose(T, 3.0)
+    assert_allclose(p, 0.11384629800665805)
+
+    T, p = stats.wilcoxon(arr, zero_method="wilcox")
+    assert_allclose(T, 1.5)
+    assert_allclose(p, 0.10247043485974937)
```

Comment:
1. where are these numbers from? 
2. please mention the issue number, gh-6070 in the comment. 
3. If the objective of the test is to make sure lists pass through correctly, you may or may not want to test specific numbers.